src/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 100px;
+  /*dont undersand why need this padding, but its required to pass tests*/
+  padding-right: 4px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1) {
```

Comment:
You don't need > selectors, it should work with just space for example
```
.stars--2 .stars__star:nth-child(-n+2)
```
https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator

Reply:
Also you could shorten those rules by not declaring them separately but rather by listing all classes for one rule
```
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```

Reply:
Good idea 👍🏼